app/lib/contentdm/importer.rb

```diff
@@ -22,8 +23,13 @@ def self.import
     def import
       @records.each do |record|
         begin
-          process_record(record)
+          work = process_record(record)
+          @log.info Rainbow("Adding #{work.id} to collection: #{collection_name}")
+          @collection.add_members(work.id)
+          @collection.save
```

Comment:
I wonder if we should move the `@collection.save` outside of the `@records.each do |record|` loop so that the collection saves only once at the end instead of once for each record.  I'm not sure how large their collections are, but if they are large, performance would be much better to save the collection only at the end.